src/tests/src/libs/antares/study/short-term-storage-input/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Useful variables definitions
+set(src_libs_antares_study "${CMAKE_SOURCE_DIR}/libs/antares/study")
+
+# ====================================
+# Tests on reading scenario-builder
+# ====================================
+set(SRC_SC_BUILDER_READ
+	short-term-storage-input.cpp
+)
+add_executable(short-term-storage-input ${SRC_SC_BUILDER_READ})
+
+target_include_directories(short-term-storage-input
+								PRIVATE
+									"${src_libs_antares_study}"
+									"${src_libs_antares_study}/parts/short-term-storage"
```

Comment:
I think it would be better to remove this line, and to `#include <antares/study.h>`. It gives you access to most elements I think (no public/private headers)